src/pure.ts

```diff
@@ -9,7 +9,7 @@ function hasDependency(name: string) {
   try {
     require(name)
     return true
-  } catch (error) {
+  } catch {
```

Comment:
No longer needed once Node 10 was dropped (actually produced a lint warning in my local environment).